packages/mobile/src/send/SendAmount.tsx

```diff
@@ -181,6 +182,7 @@ function SendAmount(props: Props) {
 
   const isAmountValid = parsedLocalAmount.isGreaterThanOrEqualTo(DOLLAR_TRANSACTION_MIN_AMOUNT)
   const isDollarBalanceSufficient = isAmountValid && newAccountBalance.isGreaterThan(0)
+  const minimumAmount = roundUp(dollarAmount.plus(estimateFeeDollars || 0), 2)
```

Comment:
`minimumAmount` needs to be added to the dependencies of `useCallback` where it's used.